app/BUILD.bazel

```diff
@@ -9,14 +9,14 @@ load("@tools_android//tools/googleservices:defs.bzl", "google_services_xml")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 
 # Source files for annotations library.
-ANNOTATION_FILES = [
+ANNOTATIONS = [
     "src/main/java/org/oppia/app/activity/ActivityScope.kt",
     "src/main/java/org/oppia/app/fragment/FragmentScope.kt",
     "src/main/java/org/oppia/app/utility/KeyboardHelper.kt",
 ]
 
 # Source files for listener library.
```

Comment:
Here & elsewhere: let's make sure it's clearly documented what each list is meant to contain. Consider the perspective of members of the team: if they are adding a new class, how do they know where to put it to avoid things breaking?

I also suggest adding brief notes here about gotchas they might run into, e.g. avoiding Kotlin synthetic imports, direct databinding view references, fragment references within layouts, etc. People will hit these issues with new code, so we should make sure that folks know how to write things to avoid breaking Bazel or needing to dig into the problem for multiple hours.

Reply:
Okay! Let me know what you think of the documentation I added!

Reply:
Followed up with some comments. Thanks for starting it!